### ISSUE TITLE
Allow users to disable tab hover cards for Chromebooks/Android tablets 

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#37632]
## Description 
The tab hover preview, as it relates to Android devices, is only available on Chromebooks or android tablets (where the browser displays the actual tabs bar). However, it appears that we've inherited the tab hover preview feature from Chromium, without a way to disable it. Users should be able to disable this feature as it can be distracting/unwanted.

There were flags previously available to do this but have since been removed (although I do not understand why). One such user has reached out to us and provided the following video showing the behavior, as well as a bug in the behavior that the user has no way out of, due to the fact that they cannot disable the feature:
https://www.dropbox.com/scl/fi/5cdciix8anesihnlg5f3o/20240417_072721.mp4?rlkey=pmxd2qq5o4lw8xjsuc5519x5c&dl=0

### Additional information

Users original post:
https://community.brave.com/t/tab-preview-pops-up-in-front-of-pages/543738/12
